app/models/claim.rb

```diff
@@ -115,6 +117,8 @@ class Claim < ApplicationRecord
   scope :submitted, -> { where.not(submitted_at: nil) }
   scope :awaiting_checking, -> { submitted.left_outer_joins(:check).where(checks: {claim_id: nil}) }
   scope :approved, -> { joins(:check).where("checks.result" => :approved) }
+  scope :approaching_check_deadline, -> { awaiting_checking.where("submitted_at < ?", CHECK_DEADLINE.ago + CHECK_DEADLINE_WARNING_POINT).where.not("submitted_at < ?", CHECK_DEADLINE.ago) }
```

Comment:
This could probably be expressed more simply with a "submitted_at < ? AND submitted_at > ?" but not a blocker.